util/detectOS.ts

```diff
@@ -1,18 +1,19 @@
+import { OS } from '@/next.constants.mjs';
 import type { UserOS } from '@/types/userOS';
 
 export const detectOsInUserAgent = (userAgent: string | undefined): UserOS => {
-  const osMatch = userAgent?.match(/(Win|Mac|Linux)/);
+  const osMatch = userAgent?.match(/(Win|Mac|Linux|AIX)/);
   switch (osMatch && osMatch[1]) {
     case 'Win':
-      return 'WIN';
+      return OS.WIN as UserOS;
```

Comment:
Instead of doing `as UserOS` here, you can cast the constant to be a `Record<UserOS, UserOS>` or simply do `as const` which should be enough,